concrete/src/Asset/CssAsset.php

```diff
@@ -175,12 +202,25 @@ public static function process($assets)
         return $assets;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @see \Concrete\Core\Asset\AssetInterface::register()
+     */
+    public function register($filename, $args, $pkg = false)
+    {
+        parent::register($filename, $args, $pkg);
+        if ($args['media']) {
```

Comment:
This throws a PHP warning if the `media` array key does not exist (see [this example](https://3v4l.org/pjvng)).

We should use `isset()` ([example](https://3v4l.org/oh6uK))
```php
if (isset($args['media']) && $args['media']) {
    //...
}
```
or `empty()` ([example](https://3v4l.org/BQvol)):
```php
if (!empty($args['media'])) {
    //...
}
```